docs/source/ttnn/tensor.rst

```diff
@@ -138,6 +138,50 @@ Storage
     The buffer of the tensor is on a ttnn device.
 
 
+Tensor Sharding
+***************
+
+Sharding refers to a tensor that is split across a distributed memory space, with that distribution ideally being abstracted away.
+Currently we support L1 sharding, which refers to the distribution of a tensor across the L1 memory space of different cores.
+
+Sharded tensors are represented in two dimensions, we compress a higher rank tensor into two-dims by compressing all upper dims in dim 0 and keeping the last dimension the same.
+For example a [1,2,3,4] tensor will be represented as a [6,4] 2D tensor before sharding.
+Data in a :class:`ttnn.Tensor` is organized in tiles, and these tiles are typically in row-major order.
+The size of the tile depends on the :class:`ttnn.Layout` and :class:`ttnn.Shape` of the tensor.
+Refer to the section about ``Layout`` to learn more.
+
+A sharded tensor's tiles are organized in sharded order on the device which we will highlight with an example.
+A tensor is only sharded on device and on the host the tiles remain in row-major order.
+Below is an example of a tensor on the host, with each page labelled.
```

Comment:
Can we create a link to page (like we have for ttnn.Layout) that explains what a page is?

Reply:
hey eon, sorry I meant a tile, which we already explain above.